app/javascript/packs/base.jsx

```diff
@@ -168,8 +168,8 @@ async function loadCreatorSettings() {
       { LogoUploadController },
       { Application },
     ] = await Promise.all([
-      import('@admin-controllers/creator_settings_controller'),
-      import('@admin-controllers/logo_upload_controller'),
+      import('@admin/controllers/creator_settings_controller'),
+      import('@admin/controllers/logo_upload_controller'),
```

Comment:
With the new alias for admin on the frontend, I needed to adjust these dynamic imports.

Reply:
#16412 covers this.